app/src/main/java/com/capstone/event_finder/network/EventRepository.java

```diff
@@ -13,10 +13,12 @@
 public class EventRepository {
 
     public EventDao eventDao;
+    public EventApi eventApi;
 
     public EventRepository(Application application) {
         EventDatabase eventDatabase = EventDatabase.getInstance(application);
         eventDao = eventDatabase.eventDao();
+        eventApi = new EventApi(this);
```

Comment:
⚠️ Like my comment in `EventApi`, we don't want the repository to have a reference to the API.  This class should specifically handle the database only.  The ViewModel should be the only class that knows about both.